modules/migration-using-must-gather.adoc

```diff
@@ -19,13 +27,33 @@ The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migra
 ----
 $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
 ----
++
+The data is saved in a local `must-gather` directory.
+
 
-. Remove authentication keys and other sensitive information.
-. Create an archive file containing the contents of the `must-gather` data directory:
+
+. Optional: Collect an expanded set of metrics data by running the `oc adm must-gather` command with the `gather_metrics_dump` script:
```

Comment:
"expanded set of metrics data" is not very specific. See my MTV update - https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.0/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#using-must-gather_mtv

Reply:
Did Derek tell you what this expanded set is?